spark_pipeline_framework/transformers/framework_fhir_meta_updater/v2/test/patient.json

```diff
@@ -0,0 +1 @@
+{"identifier":[{"period":{"start":"2020-01-01","end":"3000-01-01"},"system":"https://sources.aetna.com/patient/identifier/memberid/81","type":{"coding":[{"system":"http://terminology.hl7.org/CodeSystem/v2-0203","code":"MB"}]},"value":"300003811300"}],"address":[{"country":"US","city":"Windsor","line":["3440 Peter Street"],"postalCode":"067346","text":"3440 Peter Street, Windsor, CT  067346","state":"CT"}],"gender":"male","meta":{"profile":["http://hl7.org/fhir/us/carin/StructureDefinition/carin-bb-patient"]},"name":[{"given":["LIONEL Uddin"],"prefix":["Mr"],"text":"Mr. LIONEL Uddin STIKE M.D","family":"STIKE","suffix":["M.D"]}],"active":true,"language":"en","telecom":[{"system":"phone","value":"8765432563"}],"id":"0000000000000015116","communication":[{"language":{"coding":[{"system":"https://www.hl7.org/fhir/valueset-languages.html","code":"en"}]}}],"birthDate":"1975-04-05","resourceType":"Patient","service_slug":"cigna","client_source_url":"https://p-hi2.digitaledge.cigna.com/PatientAccess/v1-devportal"}
```

Comment:
Can we format this file, would make it easier to read

Reply:
copied from v1